kaffe/tensorflow/transformer.py

```diff
@@ -158,6 +164,25 @@ def map_eltwise(self, node):
         except KeyError:
             raise KaffeError('Unknown elementwise operation: {}'.format(op_code))
 
+    def map_reshape(self,node) :
```

Comment:
Don't see this function being used.

Reply:
oh, there's like 5 layers of invisible indirection going on ni this codebase!  It gets used though eventually.